tests/integration/queries-test.js

```diff
@@ -26,7 +26,8 @@ describe('Integration: FirebaseAdapter - Queries', function() {
 
     Ember.run(function () {
       adapter.query(store, store.modelFor('post'), query, queryArray)
-        .then(() => {
+        .then((records) => {
+          queryArray.load(records);
```

Comment:
yeah, this is definitely indicating there is some problem elsewhere. it should not be needed